presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,52 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Improve performance of queries that use decimal type. ({issue}`4886`)
+* Improve of queries involving `IN` with subqueries by extending support for dynamic
+  filtering and dynamic partition pruning. ({issue}`5017`)
+* Improve dynamic partition pruning and query performance by reducing latency of dynamic filters collection. ({issue}`4988`)
+* Improve join performance when cost-based optimizer has missing or inaccurate stats. ({issue}`5141`)
+* Enable connectors to wait for dynamic filters derived from replicated join before generating splits. ({issue}`4685`)
+* Fix query failure when lambda expression references a table column containing a dot. ({issue}`5087`)
+
+## Atop Connector Changes
+
+* Fix incorrect query results when query contains predicates on `start_time` or `end_time` column. ({issue}`5125`)
+
+## Hive Connector Changes
+
+* Allow specifying what happens if data is inserted into existing Hive partition. 
+  This can be done using `hive.insert-existing-partitions-behavior` config property. ({issue}`4999`)
+* Add property (`hive.dynamic-filtering-probe-blocking-timeout`) for delaying table scans
+  until dynamic partition pruning can be performed more efficiently. ({issue}`4991`)
+* Disable matching the existing user and group of the table or partition when creating new files on HDFS.
+  The functionality was added in 341 and is now disabled by default. You can enable it with `hive.fs.new-file-inherit-ownership`
+  configuration property. ({issue}`5187`)
+    
+## Kafka Connector Changes
+
+* Expose message headers as a `_headers` column of `map(VARCHAR, array(VARBINARY))` type. ({issue}`4462`)
+* Add write support for `TIME`, `TIME WITH TIME ZONE`, `TIMESTAMP` and `TIMESTAMP WITH TIME ZONE` 
```

Comment:
lowercase the types

Reply:
@findepi We are not super consistent in the documentation but most of the time we use uppercase. Was there a decision made to switch to lowercase?
cc: @mosabua

Reply:
i don't think there was "a switch". i know we [tried to] use[d] lowercase for presto type names in release notes before.

maybe we just need to re-decide _and_ write the decision down somewhere?

cc @electrum